.travis.yml

```diff
@@ -1,26 +1,24 @@
 language: python
 sudo: false
 
-python:
-  - "2.7"
-
-env:
-  - TOXENV=django18
-  - TOXENV=django110
-  - TOXENV=django111
-  - TOXENV=quality
+matrix:
+  include:
+    - python: '2.7'
+      env: 'TOXENV=django111'
+    - python: '3.5'
+      env: 'TOXENV=django111'
+    - python: '3.5'
+      env: 'TOXENV=quality'
 
 before_install:
-  - export DISPLAY=:99.0
-  - sh -e /etc/init.d/xvfb start
-  - sleep 3 # give xvfb some time to start
-  - wget https://github.com/mozilla/geckodriver/releases/download/v0.19.1/geckodriver-v0.19.1-linux64.tar.gz
+  - wget https://github.com/mozilla/geckodriver/releases/download/v0.25.0/geckodriver-v0.25.0-linux64.tar.gz
   - mkdir geckodriver
-  - tar -xzf geckodriver-v0.19.1-linux64.tar.gz -C geckodriver
+  - tar -xzf geckodriver-v0.25.0-linux64.tar.gz -C geckodriver
   - export PATH=$PATH:$PWD/geckodriver
+  - export BOKCHOY_HEADLESS=true
 
 addons:
-  firefox: '58.0.2'
```

Comment:
Do you need to also add the `services: -xvfb` section here: https://docs.travis-ci.com/user/gui-and-headless-browsers/#using-services

Reply:
@feanil The tests are passing without any issue, I'm not sure if we still need this service?

Reply:
Fair point, I think this is fine then.